.pre-commit-config.yaml

```
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
```

Prettier has a pre-commit hook, too. Here's the config suggestion:

```suggestion
    -   id: check-added-large-files
-   repo: https://github.com/pre-commit/mirrors-prettier
    name: Prettier
    rev: "3.2.5"
    hooks:
    -   id: prettier
        additional_dependencies:
        -   prettier@3.2.5
        -   prettier-plugin-astro@0.13.0
    	-   prettier-plugin-tailwindcss@0.5.11
```